utils/chunked_vector.hh

```diff
@@ -364,6 +366,13 @@ chunked_vector<T, max_contiguous_allocation>::chunked_vector(Iterator begin, Ite
     }
 }
 
+template <typename T, size_t max_contiguous_allocation>
+chunked_vector<T, max_contiguous_allocation>::chunked_vector(std::initializer_list<T> x)
+        : chunked_vector() {
+    reserve(x.size());
+    std::move(std::move(x).begin(), std::move(x).end(), std::back_inserter(*this));
```

Comment:
This doesn't work since initializer_list is a const container.

Reply:
good point.  will copy instead.